specs/credentialmanagement/index.src.html

```diff
@@ -927,18 +975,6 @@ <h4 id="request-credential">
       </ol>
     </li>
     <li>
-      Let <var>type</var> be lowest common ancestor interface of the types
-      referenced in <var>request</var>'s {{CredentialRequest/types}} property.
-
-      ISSUE(w3c/webappsec#289): This is terribly hand-wavey. The intent is
-      clear, but I need to do the work to walk through the list of DOMStrings
-      and convert them to interfaces and etc. Busywork for later.
```

Comment:
1. I can't speak for other browsers, but Chrome's credential manager sits on top of a relational database. Making it possible to store arbitrary data in the credential manager will be some work (and, given that we treat "credential" data separately from "browsing data", I worry a bit about the privacy implications of exposing an arbitrary data storage mechanism to the web that isn't easily cleared via the mechanisms users expect).

2. This still assumes that `LocallyStoredCredential` is what's being extended. It's not clear to me that something like `AgeVerificationCredential` is of the same kind. That is, the browser can't decide whether it has permission to grant access to the credential (or, really, access to the credential itself) without some sort of POST-based callback system that would require immediate user interaction. I don't have a good suggestion for merging that kind of model with this one. Do you?